frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -98,38 +98,74 @@ const pageDescription =
   'Catalog. Cluster admins can install additional apps which will show up here automatically.';
 
 // Filter property white list
-const filterGroups = ['kind'];
+const filterGroups = ['type'];
 
 const getAvailableFilters = (initialFilters) => {
   const filters = _.cloneDeep(initialFilters);
-  filters.kind = {
-    ClusterServiceClass: {
-      label: 'Service Class',
-      value: 'ClusterServiceClass',
+  filters.type = {
+    operatorBacked: {
+      label: 'Operator Backed',
+      active: true,
+      kinds: {
+        ClusterServiceVersion: {
+          label: 'Installed Operators',
+          value: 'InstalledOperator',
+          active: false,
+        },
+      },
+    },
+    helm: {
+      label: 'Helm Charts',
       active: false,
+      kinds: {
+        ImageStream: {
```

Comment:
This is wrong because Helm charts are not based on `ImageStreams`

Reply:
forgot to update the name

Reply:
Removed all kind groupings